src/lightning/app/cli/commands/ls.py

```diff
@@ -0,0 +1,96 @@
+import os
+from typing import List, Optional
+
+import click
+import rich
+from rich.live import Live
+from rich.spinner import Spinner
+from rich.text import Text
+
+from lightning.app.cli.commands.cd import _CD_FILE
+from lightning.app.cli.commands.connection import _LIGHTNING_CONNECTION_FOLDER
+from lightning.app.utilities.app_helpers import Logger
+from lightning.app.utilities.network import LightningClient
+
+_FOLDER_COLOR = "blue"
+_FILE_COLOR = "white"
+
+logger = Logger(__name__)
+
+
+@click.argument("path", required=False)
+def ls(path: Optional[str] = None) -> List[str]:
+    """Command to list the content of a folder from the Lightning Cloud Filesystem."""
+
+    root = "/"
+    paths = []
+
+    with Live(Spinner("point", text=Text("pending...", style="white")), transient=True) as live:
+
+        live.stop()
+
+        if not os.path.exists(_LIGHTNING_CONNECTION_FOLDER):
+            os.makedirs(_LIGHTNING_CONNECTION_FOLDER)
+
+        if not os.path.exists(_CD_FILE):
+            with open(_CD_FILE, "w") as f:
+                f.write(root + "\n")
+        else:
+            with open(_CD_FILE) as f:
+                lines = f.readlines()
+                root = lines[0].replace("\n", "")
+
+        client = LightningClient()
+        projects = client.projects_service_list_memberships()
+
+        if root == "/":
+            project_names = [_add_colors(project.name, color=_FOLDER_COLOR) for project in projects.memberships]
+            rich.print(*sorted(set(project_names)))
+            return project_names
+
+        splits = root.split("/")[1:]
```

Comment:
Could you comment what would be a typical format of `root` for a file inside a lightning app or cloudspace, for future readers

Reply:
Done !